actions/publish-github-pages/action.yml

```
@@ -1,10 +1,143 @@
 name: 'Publish GitHub Pages'
-description: 'Publish the Hugo-based CloudPosse docs website to GitHub Pages.'
+description: 'Publish the Hugo-based Cloud Posse docs website to GitHub Pages.'
+inputs:
+  content:
+    description: 'Comma-separated list of directories in the top level of the repo that contains documentation'
+    required: true
+  hugo_url:
+    description: 'URL of the Hugo site after deployment'
+    required: true
+  github_pages_push_path:
+    description: 'Location where the repo/branch that deploys to GitHub Pages will be cloned'
+    required: true
+  github_pages_pull_branch:
+    description: 'The branch of the repo which contains the documentation'
+    required: true
+  github_pages_repo:
+    description: 'Repo containing documentation to be deployed to GitHub Pages'
+    required: true
+  github_pages_push_branch:
+    description: 'The branch of the repo which GitHub Pages will deploy from'
+    required: true
+  css:
+    description: 'Custom CSS to customize the look and feel of the docs website'
+    required: false
+  config:
```

Since we're not deep merging using `yq`, then let's say these are overrides. 

```suggestion
  config_overrides:
```